scripts/sword_locales_to_json.js

```diff
@@ -0,0 +1,208 @@
+const fetchline = require('nodefetchline');
+const fs = require('fs');
+const iso6393 = require('iso-639-3');
+const i18nController = require('../app/frontend/controllers/i18n_controller.js');
+
+
+const SWORD_LANGUAGES_URL = "https://crosswire.org/svn/sword/trunk/locales.d/locales.conf";
```

Comment:
Can we instead use the files that are part of node-sword-interface?
Always fetching the latest files from CrossWire leads to un-deterministic behavior, because it is not using a fixed version of SWORD.

Reply:
> Can we instead use the files that are part of node-sword-interface?

Yes, that makes sense.

> Also I am wondering whether we should move this script to node-sword-interface in that case.
We would then simply generate an updated json file each time when there are changes in a new node-sword-interface version.

It's a nice idea. I was wondering myself if something like that should be a part of the backend. But it's nice to have a separate `json` file with all language details. 
But if we move it to `node-sword-interface` how would you suggest pulling a list of all available languages from `i18n_controller`?

Also, please note that the script is benefitting from node 14 that implements the "new" Intl API